### DISCUSSION TITLE
Add themes or labels to requested changes in PRs

### DISCUSSION THREAD

**Discussion by pierre-eliep-met** [community/community#20379] [Pull Requests]
I recently had to handle a PR with ~80 requested changes. It is quite painful to have to sort the changes in my mind, and not being able to add labels to them, or sort them by theme.
It would be nice to be able to tag requested changes, with like "formatting", "algorithmic", "naming", etc.
And then show all the comments of a certain tag, to ease review.

**Comment by gtirloni** [comment]
You should not have huge PRs. It make life harder for reviewer and for yourself.

What you're asking is better handled by a CI system that will run checks and report problems for you. If your reviewer is catching formatting errors, that's a red flag.

**Reply by pierre-eliep-met** [reply]
Agreed, the best is to have small PRs.
But there are some refactoring comments that are not catched by CI like "always use kwargs instead of positionnal args", or refactoring code propositions, etc.
Anyway, this feature would make things easier, even for small PRs, to handle similar requested changes in the meantime, which is not easy to see at first glance

**Comment by thuongtruong1009** [comment]
You should follow the common rule like [that](https://www.jeffbullas.com/the-10-new-rules-of-pr/)

**Reply by pierre-eliep-met** [reply]
I'm sorry, if it was a joke I missed it and if it was not missed it too 😇
